<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRelationWidget</class>
 <widget class="QWidget" name="EditRelationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>67</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelRelationName">
     <property name="text">
      <string>RelationName</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditDisplayLeft"/>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditDisplayRight"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
